src/utils.rs

```diff
@@ -80,7 +80,36 @@ pub fn ident_at(root: &SyntaxNode, offset: usize) -> Option<CursorInfo> {
         }
     }?;
     let parent = ident.node().parent();
-    if let Some(attr) = parent.clone().and_then(Key::cast) {
+    if let Some(node) = parent.clone().and_then(Inherit::cast) {
+        if let Some(node) = node.from() {
+            for tok in node.node().children() {
```

Comment:
The `from()` here will always return an `InheritFrom` which will always only have one child. So doing `node.inner()` instead of the raw `node.node().children()` might be preferable? _(import `rnix::types::Wrapper` for `.inner()`)_
```suggestion
            if let Some(tok) = node.inner() {
```